### ISSUE TITLE
All cards fetched using a more-than-one-player-searches ability, regardless of who owns it, will be put into the hand of the last person to search, but those cards can't be played or discarded

### ISSUE THREAD

**Issue by stash86** [PennyDreadfulMTG/modo-bugs#255]
Based on this reddit post
https://www.reddit.com/r/Pauper/comments/7pjtlm/and_another_new_bug_borderland_explorer/

**What is the expected behaviour of the card?**
Each player who discarded a card due to Borderland's ability may search his or her library for a basic land card, reveal it, put it into his or her hand, then shuffle his or her library.

**What is the actual behavior of the card?**
All lands fetched this way will end up in Borderland's controller's hand, but lands that aren't his/her can't be played or discarded, but it still counts toward total cards in his/her hand

**Screenshot**
![image](https://user-images.githubusercontent.com/31076829/37319838-26b4c7ce-2647-11e8-9a6f-d5ab9957415a.png)

![image](https://user-images.githubusercontent.com/31076829/37319864-4760a9f2-2647-11e8-95d8-e5abf9a6befd.png)

**Please provide a list of affected cards**
Affects: [Borderland Explorer], [Noble Benefactor], [Weird Harvest]

**Card Images**
 <img src="https://pennydreadfulmagic.com/image/Borderland%20Explorer|Noble%20Benefactor|Weird%20Harvest/" height="300px">

Bug Blog Text: [Borderland Explorer] and [Noble Benefactor] give both cards to the same player.

**Comment by vorpal-buildbot** [comment]
Found in bug blog.
[Borderland Explorer] and [Noble Benefactor] give both cards to the same player.
Code: [Borderland Explorer] and [Noble Benefactor] give both cards to the same player.

**Comment by vorpal-buildbot** [comment]
This bug has been removed from the bug blog!

**Comment by silasary** [comment]
Nope.  @vorpal-buildbot is misbehaving.

**Comment by BackAlleyG** [comment]
Still bugged 

Updating original comment with screenshots too

**Comment by xedoua** [comment]
Hello,

In late may I reported this bug from a Commander game when I used Noble Benefactor. It works a little differently than described and also worked with **Weird Harvest** (unmentioned so far) and Borderland Explorer.

**When in a Commander/Multiplayer Game, the last person to search with the effect got all the tutored cards.**

I did receive a comment from A wizards Customer Service Rep saying he could replicate the error above and forwarded it onto the development team for fixing. He didn't give me a time they would fix this bug by, but told me to watch the bug fixes post.
